api_client/python/timesketch_api_client/sketch_test.py

```diff
@@ -55,3 +55,25 @@ def test_get_event(self):
         self.assertIsInstance(event_data, dict)
         self.assertTrue('meta' in event_data)
         self.assertTrue('comments' in event_data['meta'])
+
+    def test_add_event_attributes(self):
```

Comment:
Could you also add a test case for `("Events need to be a list.")` ?
And another test case what happens if you add an attribute that is already present in the event?

Reply:
Added a test case for `("Events need to be a list.")` but I don't think adding one for `add an attribute that is already present in the event` will really be testing anything useful. 

Duplicate attributes won't be added and will just be noted in the `errors` field in the response which is returned directly by `timesketch_api_client.sketch.Sketch.add_event_attributes`.